bin/bin.js

```diff
@@ -208,6 +208,7 @@ const main = async () => {
     counter += 1;
     logMessage("");
   };
+  let dlCounter = 0;
```

Comment:
Could we rename this to `episodesDownloadedCounter`?

Reply:
done, see a133860